markdown/en/overview-styling-elements.md

```diff
@@ -48,11 +48,13 @@ To overwrite the default styles the following code can be used:
 
 It is important, that the style element possesses the property &quot;is&quot; with the value &quot;custom-style&quot;.
 
-## Prevent FOUC (Flash Of Unstyled Content)
-In browsers where Web Components are not natively supported you might encounter ugly flash of unstyled content on page load. 
-To prevent this we added a __remove unresolved attribute behavior__ to all elements with an visual component (e.g. ff-record-list, ff-asn and so on).
-Just annotate all elements which are shown immediately on page load with the __\[unresolved\]__  attribute. 
-In Addition add the following CSS rule on top of the page before the appearance of your  custom elements:
+## Prevent Flash Of Unstyled Content (FOUC)
+In browsers that do not yet natively support FACT-Finder Web Components you might encounter unappealing flashes of unstyled content 
+on page load. To prevent this we added a __remove unresolved attribute behavior__ to all elements with an visual 
+component (e.g. ff-record-list, ff-asn and so on). Just annotate all FACT-Finder Web Components that are shown on page load with the
+`[unresolved]` attribute. Furthermore, add the following CSS rule at the beginning of the page before the Web 
```

Comment:
I feel these `Web Components` would do better with `FACT-Finder` in front.

By the way, are these `overview-...` files actually still needed? This one is not even used anymore as far as I can see. All of them seem to have a replacement.

Reply:
We seem to have quite some files that are just legacy that hasn't been deleted yet. I'll open an issue.